package.json

```diff
@@ -131,6 +131,7 @@
     "@cypress/skip-test": "^2.6.1",
     "@emotion/babel-plugin": "^11.11.0",
     "@esbuild-plugins/node-modules-polyfill": "^0.2.2",
+    "@inquirer/prompts": "^5.2.0",
```

Comment:
Should we separate this into a separate `package.json` for the CLI? I don't want to bloat up the dependencies further. It will be compiled down into a single `cli.js` anyway.